pyEpiabm/pyEpiabm/tests/test_unit/test_routine/test_file_population_config.py

```diff
@@ -179,23 +190,34 @@ def test_print_population_loc(self, mock_write, mock_read):
         mock_write.assert_called_once()
         self.assertEqual(mock_write.call_args[0], ('output.csv',))
 
+    @patch('logging.info')
```

Comment:
```suggestion
    @patch('logging.warning')
```
(Because I changed the logging level above)